packages/core/core/src/Transformation.js

```diff
@@ -8,23 +8,22 @@ import type {
   TransformerResult,
   PackageName
 } from '@parcel/types';
+import type {WorkerApi} from '@parcel/workers';
 import type {
   Asset as AssetValue,
-  AssetRequest,
+  AssetRequestDesc,
   Config,
-  NodeId,
-  ConfigRequest,
+  ConfigRequestDesc,
   ParcelOptions
 } from './types';
-import type {WorkerApi} from '@parcel/workers';
 
 import invariant from 'assert';
 import path from 'path';
 import nullthrows from 'nullthrows';
 import {md5FromObject} from '@parcel/utils';
 
+import ConfigLoader from './ConfigLoader';
```

Comment:
I like this name a log better than `Config`

Reply:
Well... `Config` still exists. `ConfigLoader` is responsible for running plugins' `loadConfig` which takes a `Config` and uses it to store the result and all of the files that should invalidate that `loadConfig` call. I'm not super happy with the naming, but you know what they say.